lib/defs.js

```diff
@@ -73,6 +73,9 @@ function defs(name) {
       } else {
         defs[/** @type {string} */ (name)] = wkt(arguments[1]);
       }
+    } else if (typeof def === 'object' && !('projName' in def)) {
```

Comment:
I think this will cause an error if `def` is `null` or `undefined` as `typeof null = 'object'`.
```
TypeError: def is not an Object. (evaluating '"projName" in def')
```